ui/appui-layout-react/src/appui-layout-react/widget/PinToggle.tsx

```diff
@@ -54,7 +54,7 @@ export const PinToggle = React.memo(function PinToggle() { // eslint-disable-lin
       }}
       title={panelState.pinned ? unpinPanelTitle : pinPanelTitle}
     >
-      <Icon iconSpec={panelState.pinned ? <SvgUnPin /> : <SvgPin />} />
+      <Icon iconSpec={panelState.pinned ? <SvgPin /> : <SvgUnPin />} />
```

Comment:
LGTM for a backport. Should we consider rewording this back to:
`pinned ? <Unpin /> : <Pin />` OR `pinned ? <Pinned /> : <Unpinned />`

Reply:
We can, but it's probably better if these icons go into itwinui-icons and we use them from there.